www/js/diary/list.js

```diff
@@ -59,23 +61,23 @@ angular.module('emission.main.diary.list',['ui-leaflet',
 
   angular.extend($scope.defaults, Config.getMapTiles())
 
-  moment.locale('en', {
-    relativeTime : {
-        future: "in %s",
-        past:   "%s ago",
-        s:  "secs",
-        m:  "a min",
-        mm: "%d m",
-        h:  "an hr",
-        hh: "%d h",
-        d:  "a day",
-        dd: "%d days",
-        M:  "a month",
-        MM: "%d months",
-        y:  "a year",
-        yy: "%d years"
-    }
-});
+//   moment.locale('en', {
```

Comment:
Looks like this can be fixed by including the standard locales in `index.html`
https://momentjs.com/docs/#/i18n/loading-into-browser/

Reply:
I am doing it [there](https://github.com/fabmob/e-mission-phone-fabmob/blob/f8db039d97f3d403f7a1dbd16174c5f0c4423fee/www/js/main.js#L154). The main idea is that if there is no spanish (for instance) it will not use the spanish locales but the locales of the fallback language instead.

Reply:
ah, I see. I was looking at the moment documentation, 
https://momentjs.com/docs/#/i18n/loading-into-browser/
and I thought that you needed to include individual locale files - e.g.

```
<script src="locale/fr.js" charset="UTF-8"></script>
<script src="locale/pt.js" charset="UTF-8"></script>
```

But I see that you are using option B - e.g. have the combined minified locale file instead

```
     <script src="lib/moment/min/locales.min.js"></script> 
```

so that's all good.